packages/cli/src/databases/repositories/role.repository.ts

```diff
@@ -32,4 +32,11 @@ export class RoleRepository extends Repository<Role> {
 			return acc;
 		}, {});
 	}
+
+	async getIdsByNames(roleNames: RoleNames[]) {
```

Comment:
The function name doesn't imply that we are only getting IDs of roles that have `scope = workflow`. Maybe it could be included in it? E.g. something like `getIdsInScopeWorkflowByNames`?

Reply:
Renamed! Unfortunately this happens very often with all these PRs - the existing logic is too coupled to the callsite, so the new repository method ends up being overly specific and would require more refactoring to be truly reusable.